src/Select/index.tsx

```diff
@@ -96,7 +136,7 @@ export const isSelectedItemInCategory = (selectedItem, categoryChildren) => {
  * @param {String} userInput most recent thing a user typed while focused on input
  * @returns {String} the string to use for typeahead for each given `selectItem`
  */
-// eslint-disable-next-line no-unused-vars
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const defaultGetTypeAheadString = (userInput = "", selectItem) => {
```

Comment:
I'm surprised it doesn't complain about this, it seems odd that a non-optional parameter comes after an optional one? Are the items in the wrong order, the docs suggest it should be `selectItem` first?